src/pages/events/components/constants.js

```diff
@@ -0,0 +1,27 @@
+import { EVENTS_22_23 } from "assets/index"
+
+const PAST_EVENTS = 
+{
+    "2022/2023": [
+        { image: EVENTS_22_23[0], title: "SEA Pit Night", date: "Wednesday, April 8 | 9PM-1AM", loc: "The Pit Pub @ UBC" },
+        { image: EVENTS_22_23[1], title: "GALA(U) — Gala With GISAU", date: "Friday, March 31 | 6-8PM", loc: "AMS Nest Great Hall South" },
+        { image: EVENTS_22_23[2], title: "GISAU Annual General Meeting", date: "Friday, March 24 | 6.15PM", loc: "AMS Nest Room 2306/2309" },
+        { image: EVENTS_22_23[3], title: "GISAU’s Study Cafe", date: "Wednesday, March 15 | 12-6PM", loc: "Abdul Ladha Room 103/104" },
+        { image: EVENTS_22_23[4], title: "FIESTA Vancouver", date: "Saturday, March 4 | 4-8PM", loc: "Nikkei National Museum & Cultural Centre" },
+        { image: EVENTS_22_23[5], title: "Indomie-t Me at the Finish Line", date: "Friday, February 17 | 6.30-8.30PM", loc: "AMS Nest Lower Atrium" },
+        { image: EVENTS_22_23[6], title: "Welcome Back Picnic", date: "Monday, January 30 | 1-3PM", loc: "Lev Bukhman Theatre Lounge @ UBC AMS Nest" },
+        { image: EVENTS_22_23[7], title: "GISAU’s Study Cafe", date: "Tuesday, December 6 | 1-7PM", loc: "Abdul Ladha Room 103/104" },
+        { image: EVENTS_22_23[8], title: "Taste of SEA", date: "Monday, November 14 | 6.30-9PM", loc: "Great Hall South, AMS Nest" },
+        { image: EVENTS_22_23[9], title: "Rumah Hantu", date: "Friday, November 4 | 1-8.30PM", loc: "AMS Nest Room 2314" },
+        { image: EVENTS_22_23[10], title: "Liwetan", date: "Saturday, October 15 | 5-7.30PM", loc: "Consulate General of Indonesia" },
+        { image: EVENTS_22_23[11], title: "Halte: UBC", date: "Thursday, September 29 | 6-9PM", loc: "AMS Nest Great Hall South" },
+        { image: EVENTS_22_23[12], title: "Opt to Know Co-op", date: "Friday, August 26 | 8PM", loc: "via Zoom" },
+        { image: EVENTS_22_23[13], title: "Independence Day Picnic", date: "Saturday, August 20 | 4.30PM", loc: "Nobel Park @ UBC" },
+    ],
+    "2021/2022": [],
```

Comment:
I think it's better to just remove 2018-2022 because I don't think you will put any in them. Unless they somehow wanted put past events from previous years (I feel like it will be hard for them to collect the data).

Reply:
I was thinking, we can just scroll through our instagram page and gather all the info from there. Just like we did for 2022/2023 events. Maybe not from 2018, probably 2020-now

Reply:
Well if you want to do that, then go ahead